build/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/ash
+INTERNAL_IP=""
+DOMAIN=""
+echo "Trying to fetch domain and internal IP..."
+for i in {1..30}; do
+    if [ -z $INTERNAL_IP  ]; then
+        INTERNAL_IP=$(curl -s http://my.dappnode/global-envs/INTERNAL_IP)
+    fi
+
+    if [ -z $DOMAIN ]; then
+        DOMAIN=$(curl -s http://my.dappnode/global-envs/DOMAIN)
+    fi
+
+    if [ -z $INTERNAL_IP ] || [ -z $DOMAIN ]; then
+        echo "Retrying in one second..."
+        sleep 1
+    else
+        echo "Done, found $DOMAIN, $INTERNAL_IP"
+        break
+    fi
+done
```

Comment:
While this loop is going what would actions could be affected? Is it possible to start dnscrypt immediately and reboot it latter when the `$DOMAIN, $INTERNAL_IP` is available?

Reply:
This is only necessary for cloaking rules (overriding xxxxxx.dydns.dappnode.io resolution) and is not critical for general functionality of resolver. I am not sure whether dns-crypt even needs to be restarted when updating these files. But if it has to be restarted, we would have to figure out way to detect when that becomes available and restart it. But, I don't that would be particularly difficult.